### PR TITLE
data: khairalanam

### PR DESCRIPTION
**Pull request by khairalanam**

### CODE REVIEW

File: data/khairalanam.json [hunk 1]
```diff
@@ -0,0 +1,39 @@
+{
+  "name": "Khair Alanam",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Creative Web developer, UIUX Designer, Exploring tech in general, plays Minecraft",
+  "avatar": "https://github.com/khairalanam.png",
```
Comment (SaraJaoude) [inline data/khairalanam.json:6]: the following sections are no longer needed:
```suggestion
  "bio": "Creative Web developer, UIUX Designer, Exploring tech in general, plays Minecraft",
```

File: data/khairalanam.json [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+    {
+      "name": "Check out my posts on Devto",
+      "url": "https://dev.to/khair_al_anam",
+      "icon": "FaDev"
+    },
+    {
+      "name": "Check out my designs on Dribbble",
+      "url": "https://dribbble.com/khair_al_anam",
+      "icon": "FaDribbble"
+    },
```
Comment (SaraJaoude) [inline data/khairalanam.json:34]: remove comma:
```suggestion
    }
```

**Review by SaraJaoude — APPROVED** [review]
looks good

File: data/khairalanam.json (hunk 1 above)
Comment (khairalanam) [reply data/khairalanam.json]: Okay noted.

**Comment by khairalanam** [comment]
Thank you so much!

**Pull request merged**